### PR TITLE
Increase exclusion list.

### PR DESCRIPTION
**Pull request by EdOverflow**
Since you are using a blacklist approach, let's make absolutely sure no address can bypass it.

### CODE REVIEW

File: lib/private_address_check.rb [hunk 1]
```diff
@@ -6,21 +6,37 @@
 module PrivateAddressCheck
   module_function
 
+module PrivateAddressCheck
```
Comment (reedloden) [inline lib/private_address_check.rb]: Incorrectly added?

File: lib/private_address_check.rb [hunk 2]
```diff
@@ -6,21 +6,37 @@
...
+  module_function
+
   CIDR_LIST = [
-    # Loopback
```
Comment (reedloden) [inline lib/private_address_check.rb]: Please keep comments and segment each addition into specific sections under a comment that explains what it is.

**Review by reedloden — CHANGES_REQUESTED** [review]

File: lib/private_address_check.rb (hunk 2 above)
Comment (EdOverflow) [reply lib/private_address_check.rb]: @reedloden I plan on adding comments in just a bit. On top of that, this PR doesn't even include tests so it isn't ready for merging quite yet.

**Comment by reedloden** [comment]
Thoughts, @jtdowney?

**Comment by jtdowney** [comment]
Sorry for the delay, for some reason I wasn't getting any notifications from GitHub for this repo. I think this looks good. My only feedback is that the quotes are inconsistent. The rest of the gem uses double quotes so it would be good to do that here.

**Comment by EdOverflow** [comment]
@jtdowney That should be fixed now.

**Comment by jtdowney** [comment]
Thanks!

**Comment by reedloden** [comment]
This is [CVE-2017-0909](https://cve.mitre.org/cgi-bin/cvename.cgi?name=2017-0909).

**Pull request merged**